plugins/available/base.plugin.bash

```diff
@@ -1,6 +1,14 @@
 cite about-plugin
 about-plugin 'miscellaneous tools'
 
+
+function mkdirr() {
```

Comment:
Can you please add the missing function headers to both of your new functions? Please take a look at the `down4me` function a couple of lines down.

You'd probably need to add the `about`, `param`, `example` and `group` headers to both functions.